src/guc.c

```diff
@@ -1207,6 +1209,18 @@ _guc_init(void)
 							 NULL,
 							 NULL);
 
+	DefineCustomBoolVariable(MAKE_EXTOPTION("enable_partitioned_hypertables"),
```

Comment:
Should we protect this with TS_DEBUG even? We dont want this to be used for serious stuff while it is in development